CHANGES.rst

```diff
@@ -4,6 +4,12 @@ Latest changes
 In development
 --------------
 
+- Make `joblib.Memory` caching system compatible with `numpy >= 1.20.0`.
+  In the future, users updating `joblib` to a release that includes this
+  fix will see their previous cache invalidated (and thus wiped out) if they
```

Comment:
```suggestion
  fix will see their previous cache invalidated if they
```

The cache is not thus wiped out, the old entries are still on the disk.

Reply:
You're right, thanks.